tests/Unit/EventTest.php

```diff
@@ -154,6 +154,50 @@ public function test_cron_life_time(): void
         );
     }
 
+    public function test_cron_life_time_get_from(): void
```

Comment:
Almost there :)
- name test cases like `test_get_from` and `test_get_to`
- test `between` in other test case
- use data provider (can be one for both cases) to test `string` and `\DateTime` in `from`/`to`
- use camelCase for variables' names
- use `assertSame` in cases like `self::assertTrue($date_from === $event->getFrom());`

Reply:
Test cases renamed.
Test case between introduced.
Variables' name renamed in camelCase.

Do you prefer to have the dataproviders all together at the end of the controls file near the other one or near the functions that use them?

Reply:
> Do you prefer to have the dataproviders all together at the end of the controls file near the other one or near the functions that use them?

End of file, before private methods, please.